server/app.js

```diff
@@ -30,6 +29,7 @@ app.use(express.json());
 
 //routers
 app.use(blogRoutes);
```

Comment:
Great work @sebastien15. As feedback, I think it would be better if you create a single index.js inside the routes folder and import all routes in that file. Then after you import that file in this app.js file. That will make your codes cleaner.

Reply:
created one file for all routes and imported it in app.jsJ